### DISCUSSION TITLE
Favicon Generator

### DISCUSSION THREAD

**Discussion by Noxis73** [CorentinTh/it-tools#1353] [Ideas]
I think the title says all. Would be very handy to have a good favicon generator in place. 
By the way: Thanks for your wonderful tools-collection!

**Comment by gitmotion** [comment]
PR: https://github.com/CorentinTh/it-tools/pull/1361
Created an all purpose image resizer which also includes .ico as a file format for upload/download
- preview here: https://gitmotion-it-tools-beta.vercel.app/image-resizer
- image resizer with customizable dimensions
- bases resizes on the original upload on each update
- accepts and downloads as: .jpg, .jpeg, .png, .bmp, .ico, .svg
- responsive and stays with card (with scrollbars) when dimensions are large
- retains filename with dimensions appended
![demo](https://github.com/user-attachments/assets/1bd67960-34df-4de3-882c-ceca46e46a0a)

<details>

<summary>Want this change?</summary>

Check it out here: (https://gitmotion-it-tools-beta.vercel.app/image-resizer)

You can use my image in your docker-compose file if you want this functionality along with my other additions until the main branch has been updated. [view full list of features added in readme](https://github.com/gitmotion/it-tools/tree/gitmotion/main)

## Installation methods

| Docker Image                            | Local Installation                                                                                                          |
|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
| <span title="triple click me!">`ghcr.io/gitmotion/it-tools:latest`</span> | <span title="triple click me!">`git clone -b gitmotion/main https://github.com/gitmotion/it-tools.git && cd it-tools/ && pnpm i && pnpm dev`</span> |
| replace your current image with this image | copy & paste oneliner (from github repo) |
| You may need to clear cache and hard reload to get new features loading | Installing packages for the first time may take some time; please wait until it finishes |

</details>

**Comment by gitmotion** [comment]
@Noxis73 
since PRs are a bit backed up you can use this image in your docker-compose file if you want this functionality along with my other additions as well until the main branch has been updated:

**docker image:**
```
ghcr.io/gitmotion/it-tools:latest
```
- you may need to clear cache and hard reload to get new features loading

#### OR locally
**from github repo (copy & paste oneliner):**
```
git clone -b gitmotion/main https://github.com/gitmotion/it-tools.git && cd it-tools/ && pnpm i && pnpm dev
``` 
- (installing packages for the first time may take some time but wait till it finishes)

**Reply by Noxis73** [reply]
wow, thanks for your effort! 👍

**Comment by sharevb** [comment]
Hi @gitmotion and @Noxis73 , I also integrated @gitmotion and othrer PR, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
